bitcoin/esplora/nginx.conf

```diff
@@ -0,0 +1,10 @@
+server {
+
+  listen 80;
+
+  location / {
+    root     /usr/share/nginx/html;
+    index    index.html index.htm;
```

Comment:
I see that https://stackoverflow.com/questions/56213079/404-error-on-page-refresh-with-angular-7-nginx-and-docker uses the same `index.htm`. Do you know if we need both `.html` and `.htm`?

Reply:
Yeah, we probably don't need it as we expect to generate `index.html` into `dist`.